README.rst

```diff
@@ -34,11 +34,28 @@ PyDAX (Under Development)
    :target: https://github.com/CODAIT/pydax/commit/master
    :alt: Development Environment
 
-PyDAX is a Python API that enables easy, pragmatic, and elegant programmatical downloading and loading of datasets.
+PyDAX is a Python API that enables data consumers and distributors to easily use and share datasets, and establishes a
+standard for exchanging data assets. It enables:
+
+- a data scientist to have a simpler and more unified way to begin working with a wide range of datasets, and
+- a data distributor to have a consistent, safe, and open source way to share my datasets with interested communities.
+
+.. sidebar:: Quick Example
```

Comment:
The sidebar formatting looks really weird to me. Is there a way to make this look nicer?

![image](https://user-images.githubusercontent.com/54918401/105923777-dab27500-600a-11eb-82f3-61a0fe5cd9f1.png)

Reply:
Do we really need this example on top of the Quick Start section? It seems it is repeating content from the Quick Start section, which feels redundant.

Reply:
I think quickstart section is too long to get someone with technical background to be interested in 2min. The most important lines seem to be buried inside the other features.

The formatting is decided by the theme and the rst compiler and we can't do much about it. We can remove the box entirely though, if that looks better.

Reply:
Is there a setting that controls page width? It seems the problem is the page content area is too narrow squeezing the two code cells into using horizontal scroll bars.

Reply:
I don't see the option in the theme: https://sphinx-rtd-theme.readthedocs.io/en/stable/configuring.html

So I think we have to add a custom CSS

Reply:
I've added a custom css

Reply:
Thanks, that looks better